main_adapter.py

```diff
@@ -93,14 +129,15 @@ def predict(self, batch, **kwargs):
                 if len(system_prompt) > 0:
                     messages.append({"role": "system",
                                      "content": system_prompt})
-                messages.extend([{"role": "assistant",
-                                  "content": context},
-                                 {"role": "user",
-                                  "content": question}])
+                messages.extend([
+                    {"role": "user",
```

Comment:
this is what we discussed about the assistant role, right?

Reply:
it is not 100% that, 
this is how the nvidia models work, there will be a different refactor

Reply:
in call_coding_model you don't use stream?